### ISSUE TITLE
Vendor css files are not concatenated.

### ISSUE THREAD

**Issue by nterray** [ngbp/ngbp#275]
In `build.config.js` I have this:

``` javascript
vendor_files: {
  js: […],
  css: [
    'vendor/angular-ui-tree/dist/angular-ui-tree.min.css'
  ],
  …
}
```

When I run `grunt` command, the vendor files are not concatenated into the resulting css file. Browsing previous related issues showed me that vendor files are listed in the `index.html`; however I do not use this index.html file.

My application is bootstraped in a php-generated html file and I do not want to update the php part every time I include a vendor css file.

How can I force the concatenation?

By the way, what is the point to not concatenate vendor css file even when using the generated `index.html` file?

**Comment by bretongoers** [comment]
I ran into the same issue. Instead of mucking around in the Gruntfile, I gave up and copied all the CSS from any vendor CSS files directly into my LESS files. At that point it's no longer required to even include the vendor css in the 'vendor_files.css'

**Comment by nterray** [comment]
That do not seems to be an ideal solution as you have to think about it
each time you upgrade the library or you add a new one. Anyway thanks for
the tip!

Glad to see that someone else encounters the same issue.

**Comment by garris** [comment]
I addressed this in [pull request 305](https://github.com/ngbp/ngbp/pull/305).  Looks like css files should be passed straight through -- not concatenated. (I gathered this since index.html is calling any named css files as a dependency).  Seems there could be situations where you would want this un-concatenated option, and maybe others where you'd want the option to bake these into the concat file.   It will be interesting to see how this is treated in the warlock version.

**Comment by lowe0292** [comment]
@garris thanks for the solution.

I know it's not sexy, but I got my vendor css to compile by listing them in `concat.build_css` like so:

```
        concat: {
            // The 'build_css' target concatenates compiled CSS and vendor CSS together.
            build_css: {
                src: [
                    '<%= vendor_files.css %>',
                    '<%= recess.build.dest %>',
                    'vendor/angular-loading-bar/src/loading-bar.css'
                ],
                dest: '<%= recess.build.dest %>'
            },
            ...
```

Hope it helps someone. :]

**Comment by yaronguez** [comment]
I was able to resolve this by adding the concat:build_css task step to the "compile" task.  Previously this was only in the "build" task.  The new compile line looks like the following:

```
grunt.registerTask( 'compile', [
   'less:compile', 'concat:build_css', 'copy:compile_assets', 'ngmin', 'concat:compile_js', 'uglify',    'index:compile'
]);
```

**Comment by Merott** [comment]
Fixed with 67739b8.
